standards/core/rendezvous.md

```diff
@@ -1,74 +1,124 @@
+
+By combining rendezvous with
 [Discv5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md#node-discovery-protocol-v5) and
 [34/WAKU2-PEER-EXCHANGE](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/34/peer-exchange.md#abstract),
-Waku nodes will reach a good number of meaningful peers
-faster than through a single discovery method.
+Waku nodes can more quickly reach a meaningful set of peers
+than by relying on a single discovery method.
 
 ## Semantics
-Waku rendezvous fully inherits the [libp2p rendezvous semantics](https://github.com/libp2p/specs/blob/master/rendezvous/README.md#rendezvous-protocol).
+
+Waku rendezvous extends the [libp2p rendezvous semantics](https://github.com/libp2p/specs/blob/master/rendezvous/README.md#rendezvous-protocol) by using `WakuPeerRecord` instead of the standard libp2p `PeerRecord`.
+This allows nodes to advertise additional Waku-specific metadata beyond what is available in the standard libp2p peer record.
 
 ## Specifications
-The namespaces used to register and request MUST be in the format `rs/cluster-id/shard`.
-Refer to [RELAY-SHARDING](https://github.com/waku-org/specs/blob/master/standards/core/relay-sharding.md) for cluster and shard information.
+
+**Libp2p Protocol identifier**: `/vac/waku/rendezvous/1.0.0`
+
+### Namespace Format
+
+The namespaces used to register and request MUST be in the format `rs/<cluster-id>/<capability>`.
+
+This allows for discovering peers with specific capabilities within a given cluster.
+Currently, this is used for Mix protocol discovery where the capability field specifies `mix`.
+
+Refer to [RELAY-SHARDING](https://github.com/waku-org/specs/blob/master/standards/core/relay-sharding.md) for cluster information.
+
+### Registration and Discovery
 
 Every [Waku Relay](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/11/relay.md) node SHOULD be initialized as a rendezvous point.
 
 Each relay node that participates in discovery
 MUST register with random rendezvous points at regular intervals.
 
-We RECOMMEND a registration interval of  10 seconds.
-The node SHOULD register once for every shard it supports,
-registering only the namespace corresponding to that shard.
+All relay nodes participating in rendezvous discovery SHOULD advertise their information using `WakuPeerRecord`. For nodes supporting the Mix protocol, the `mix_public_key` field MUST be included. For non-Mix relay nodes, the `mix_public_key` field SHOULD be omitted. The standard libp2p PeerRecord is not used for Waku rendezvous; all advertised records MUST conform to the `WakuPeerRecord` definition.
```

Comment:
At this point `mix_public_key` has not been introduced. Perhaps we can have a `Wire protocol section` preceding this and the previous paragraph (before "Namespace format") that first introduces the `protobuf` structure so that this discussion is in context?